docs/examples/container-config.yaml

```diff
@@ -1,7 +1,7 @@
 metadata:
   name: busybox
 image:
-  image: busybox:latest
+  image: index.docker.io/busybox:latest
```

Comment:
could you remove `index` here?

Reply:
@feiskyer done here as well, sorry it missed that 2nd one initially, and sorry it took a moment to get back here. Merge now?